lib/spreewald/web_steps.rb

```diff
@@ -195,16 +195,13 @@
 # Checks for the existance of an input field (given its id or label)
 Then /^I should( not)? see a field "([^"]*)"$/ do |negate, name|
   expectation = negate ? :not_to : :to
-  patiently do
-    begin
-      # In old Capybaras find returns nil, so we assign it to `field`
-      field = find_with_disabled(:field, name)
-    rescue Capybara::ElementNotFound
-      # In Capybara 0.4+ #find raises an error instead of returning nil
-      # We must explicitely reset the field variable from a previous patiently iteration
-      field = nil
+
+  if negate
+    expect(page).to have_no_field(name)
+  else
+    patiently do
```

Comment:
Probably not as `have_no_field` will return directly. I just thought there is never a need to wait in this case, so I was focusing on the positive part of the query. Should we change it?

Reply:
I have no strong opinion on that, but most other steps do it this way.

(Sidenote: Looks like other steps could also need a refactoring anytime, because they aren't using the capybara matchers.)